setup.py

```diff
@@ -908,9 +908,13 @@ def create_linux_bundle_gunk(ddir: str, libdir_name: str) -> None:
         run_tool([make, 'docs'])
     copy_man_pages(ddir)
     copy_html_docs(ddir)
-    icdir = os.path.join(ddir, 'share', 'icons', 'hicolor', '256x256', 'apps')
-    safe_makedirs(icdir)
-    shutil.copy2('logo/kitty.png', icdir)
+    for sz in (16, 32, 64, 128, 256, 512, 1024):
+        icdir = os.path.join(ddir, 'share', 'icons', 'hicolor', f'{sz}x{sz}', 'apps')
+        safe_makedirs(icdir)
+        shutil.copy2(f'logo/kitty-{sz}.png', f'{icdir}/kitty.png')
+    icscaldir = os.path.join(ddir, 'share', 'icons', 'hicolor', 'scalable', 'apps')
+    safe_makedirs(icscaldir)
+    shutil.copy2('logo/kitty.svg', icscaldir)
```

Comment:
if the other changes are too much then just this would suffice

Reply:
Yes, I would prefer just installing the svg icon rather than having so many mostly useless PNG ones. Guess I am a little OCD :)